dataactbroker/handlers/accountHandler.py

```diff
@@ -584,9 +586,12 @@ def list_users(self):
     def list_user_emails(self):
         """ List user names and emails """
         sess = GlobalDB.db().session
-        user = sess.query(User).filter(User.user_id == LoginSession.getName(flaskSession)).one()
         try:
-            users = sess.query(User).filter(User.cgac_code == user.cgac_code, User.user_status_id == USER_STATUS_DICT["approved"], User.is_active == True).all()
+            users = sess.query(User).filter_by(
+                cgac_code=g.user.cgac_code,
+                user_status_id=USER_STATUS_DICT["approved"],
```

Comment:
eventually, we'll need to move away from this `user_status` since we're not handling registration/approval in the broker anymore.